docs/toc.html

```diff
@@ -106,6 +106,15 @@
                         <li><a href="#ext4">Ext4 Notes</a>
                     </ul>
                 <li><a href="#monitoring">6.6 Monitoring</a>
+                    <ul>
+                        <li><a href="#selector_monitoring">Selector Monitoring</a></li>
+                        <li><a href="#common_node_monitoring">Common Node Monitoring</a></li>
+                        <li><a href="#producer_monitoring">Producer Monitoring</a></li>
+                        <li><a href="#new_consumer_monitoring">New Consumer Monitoring</a></li>
+                        <li><a href="#connect_monitoring">Connect Monitoring</a></li>
+                        <li><a href="#kafka_streams_monitoring">Streams Monitoring</a></li>
+                        <li><a href="#others_monitoring">Others</a></li>
```

Comment:
In ops.html the `basic_ops_audit` section is in Monitoring. Do you think we should include that as well?

Reply:
Thanks for the review! 

I noticed that section as well and I think that section could use some refactoring/rewording. The mentioned JIRA [KAFKA-260](https://issues.apache.org/jira/browse/KAFKA-260) is very old from 0.8.0. The `audit` section mentions features similar to a data loss auditing pipeline, but it looks like it was never implemented in the end ("Won't Fix"). I believe what came out of that JIRA issue eventually was message timestamps.

Regardless, that section implies that Kafka itself provides such auditing features, but these features must be built on top of Kafka (by leveraging timestamps for example).

I believe we should change the wording to give a recommendation like the `Others` section, and include it to the ToC. Alternatively, we could remove the section altogether.

What are your thoughts? @viktorsomogyi @guozhangwang

Reply:
Since the audit feature is not part of Kafka, I think we can remove the audit section.